app/assets/stylesheets/user-styles.css

```diff
@@ -28,10 +28,10 @@ img.right {
   margin: -8px; padding: 8px;
 }
 
-#sidebar .sidebar-title {
+#sidebar .sidebar_title {
```

Comment:
@ilterates We're using `.sidebar-title` in a lot of places in this app.  We can't remove it!